editor/index.html

```diff
@@ -378,7 +378,10 @@
       }, false );
 
       document.addEventListener( 'keydown', function ( event ) {
-
+	  // Visualizer to ignore key combinations using "ctrl" to avoid breaking GUI hot-keys
+	  if ( event.ctrlKey)
+	  	return;
+	  	
```

Comment:
@aymanhab can you clarify the syntax here? Where is the closing `}`? I think maybe the identation is not consistent.

Reply:
@chrisdembia yes the indentation is inconsistent because we use spaces everywhere (and our editors setup this way) but the threejs code base uses tabs.
There are no braces since it's one line statement inserted in the middle of a block.